armi/operators/operator.py

```diff
@@ -618,6 +631,49 @@ def interactAllCoupled(self, coupledIteration):
         activeInterfaces = [ii for ii in self.interfaces if ii.enabled()]
         self._interactAll("Coupled", activeInterfaces, coupledIteration)
 
+    def _tightCouplingOldValues(self):
+        activeInterfaces = [ii for ii in self.interfaces if ii.enabled()]
+        for interface in activeInterfaces:
+            interface.tightCouplingOldValue = interface.getTightCouplingValue()
```

Comment:
this could actually be moved into the interface `Interface` class and called during the `interactCoupled` hook. then you wouldn't have to check enabled a 2nd time or anything, and you'd also avoid filling up this class as @john-science  says.

Reply:
This method has been modified and moved into the `o.interactAllCoupled` method. To address your previous comment about the verbiage on the method name, we moved the action of retrieving and storing the previous iteration value to the `interface.TightCoupler` class. The following is a snippet of the `o.interactAllCoupled` method from commit https://github.com/terrapower/armi/pull/1033/commits/e01ecd0f21ec6d2911cf6485e633dd09646f8c5e
![image](https://user-images.githubusercontent.com/18561046/210875189-73d53bd2-5eeb-44fb-88ae-52fc8a3194a0.png)